west.yml

```diff
@@ -129,7 +129,7 @@ manifest:
       groups:
         - hal
     - name: hal_stm32
-      revision: c36136b253a46ed8dd63525eeee659393047ce83
+      revision: 7021c43bf97d075808470b7f63da9e33ce7c01a4
```

Comment:
Please use the following for now.
```
revision: pull/161/head
```
Once HAL PR is merged, we'll use the definitive SHA1

Reply:
Neat, thanks!